tests/test_prod_smoketest.py

```diff
@@ -24,7 +24,7 @@ class ProdSmoketest(BaseSmokeTest):
     def setUpClass(cls):
         super().setUpClass()
         cls.url = "https://www.example.com"
-        cls.query = {"query": {"bool": {"must": [{"term": {"admin_deleted": "true"}}]}}}
+        cls.query = {"query": {"bool": {"must_not": [{"term": {"admin_deleted": "true"}}]}}}
```

Comment:
Won't this change interfere with the subscription notification testing?

We may need to define two queries.

Reply:
Due to the test not creating/destroying data, it only checks if the subscription can successfully be created/retrieved. From my tests locally it works successfully with the new query. I can add in the older one if requested.  

![Screen Shot 2019-04-30 at 12 46 36 PM](https://user-images.githubusercontent.com/1947534/56988993-37d37b00-6b46-11e9-9c06-7eaa4e3c9e0c.png)

Reply:
OK, I understand.

Please make it clear (via comment or naming scheme) that the subscription query is not expected to be used.

Reply:
added comment